### ISSUE TITLE
No site assignment selection for import

### ISSUE THREAD

**Issue by sharonmleon** [omeka-s-modules/Omeka2Importer#117]
Seems like items are being default assigned to all sites.

**Comment by zerocrates** [comment]
It's probably, should be, assigning to all the ones that mark "add all new items to this site".

But yes we should apply the same stuff from the other importers here.

**Comment by sharonmleon** [comment]
Yes, that's definitely it. Odd that the option was checked for all the sites, but it was.
